torch/quantization/fx/prepare.py

```diff
@@ -412,6 +419,44 @@ def maybe_insert_input_observers_for_node(
     node.args = tuple(new_args)
     node.kwargs = new_kwargs  # type: ignore[assignment]
 
+def maybe_insert_input_equalization_observers_for_node(
```

Comment:
is equalization only supported for `nn.Linear` and `F.linear` nodes, or is it going to be supported more generically for more types of nodes?

If it's only linears, might make sense to write this function specifically for linears instead of making it generic.

Reply:
I think in the future we hope to support equalization for more types of nodes (ex. conv) but I think as of right now, it only supports linear nodes.

Reply:
in that case, it would be good to have something like

```
# in equalization specific config file
def node_supports_equalization(node: Node) -> bool

# in prepare.py, in function which inserts equalization observers
if not node_supports_equalization(node):
    return
... continue with inserting equalization observers...
```

Reply:
If I'm understanding this correctly, is the purpose to remove the passing of the `equalization_qconfig_dict` and setting it as a global variable in `_equalize.py` so that we can abstract the checking of if the node supports equalization?

Right now, we check to see if the node supports equalization by seeing if it exists in `equalization_qconfig_map`, so does that mean that instead of doing this, we would move all logic to `node_supports_equalization`?

Reply:
I thought `equalization_qconfig_map` just checks what the user asked for.  Does it actually do any checking whether what the user asked for is supported?

Reply:
Sorry for the late response!! I think I understand what you mean. For now, we would checking for if the node is an `nn.Linear` or `F.linear`?